src/comm/AcceptLimiter.h

```diff
@@ -56,7 +56,7 @@ class AcceptLimiter
     static AcceptLimiter Instance_;
 
     /** FIFO queue */
```

Comment:
This comment needs to be updated. Set are not ordered so at the very least this cannot be a FIFO.

Reply:
I've removed the comment entirely. I was trying to think of something to replace it with, but "set of deferred TcpAcceptors" seemed a bit redundant.